test/25-dumper-fuzzy.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+
+var path = require('path');
+var fs   = require('fs');
+
+
+suite('Properties', function () {
+  var directory = path.resolve(__dirname, 'properties');
+
+  fs.readdirSync(directory).forEach(function (file) {
```

Comment:
Could you put here content of `./properties/identity.js` instead of files load? You do too complicated load for single test that will never be extended (with high probability).